api/v1beta1/azurecluster_validation.go

```diff
@@ -498,3 +508,12 @@ func validateCloudProviderConfigOverrides(old, new *CloudProviderConfigOverrides
 	}
 	return allErrs
 }
+
+func validateOutboundIP(frontendIP FrontendIP) error {
+	outboundIP := net.ParseIP(frontendIP.PublicIP.OutboundIP)
+	_, _, err := net.ParseCIDR(frontendIP.PublicIP.OutboundIP)
+	if outboundIP == nil && err != nil {
+		return err
+	}
+	return nil
```

Comment:
wdyt about this?
```suggestion
	outboundIP := net.ParseIP(frontendIP.PublicIP.OutboundIP)
	if outboundIP != nil {
	   return nil
	}
	
	_, _, err := net.ParseCIDR(frontendIP.PublicIP.OutboundIP)
	if err == nil {
	   return nil
	}
	
	return errors.Wrap("could not parse into an ip or prefix", err)
```

Reply:
or atleast wrap the error with information about `outboundIP == nil` condition

Reply:
sounds fair to me